src/components/PeopleTable.tsx

```diff
@@ -0,0 +1,65 @@
+import React from "react"
+import { Person } from "../types"
+import { PersonLink } from "./PersonLink"
+import classNames from "classnames"
+
+type Props = {
+  people: Person[];
+  selectedSlug: string | undefined;
+}
+
+export const PeopleTable: React.FC<Props> = ({ people, selectedSlug }) => {
+  if (people.length === 0) {
+    return (
+      <p data-cy="noPeopleMessage">There are no people on the server</p>
+    );
+  }
+
+  return (
+    <table
+          data-cy="peopleTable"
+          className="table is-striped is-hoverable is-narrow is-fullwidth"
+        >
+          <thead>
+            <tr>
+              <th>Name</th>
+              <th>Sex</th>
+              <th>Born</th>
+              <th>Died</th>
+              <th>Mother</th>
+              <th>Father</th>
+            </tr>
+          </thead>
+
+          <tbody>
+            {people.map(person => {
+              const getParentElement = (parentName: string | null) => {
```

Comment:
Can we move this logic at first load?

I meant to use something like `preaparedPeople`